marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/MintBurn.hs

```diff
@@ -59,8 +62,17 @@ tests = testGroup "MintBurn"
       "Mint events created and indexed are returned when querying the indexer"
       "queryMintedValues" queryMintedValues
   , testPropertyNamed
-      "Querying a resumed indexer returns only the persisted events"
-      "resume" resume
+      "Querying a recreated indexer should only the persisted events, and not the in-memory events of the initial indexer"
+      "propRecreatingIndexerFromDiskShouldOnlyReturnPersistedEvents "
+      propRecreatingIndexerFromDiskShouldOnlyReturnPersistedEvents
+  , testPropertyNamed
+      "The points the indexer can be resumed from should return at least the genesis point"
+      "propResumingShouldReturnAtLeastTheGenesisPoint"
+      propResumingShouldReturnAtLeastTheGenesisPoint
```

Comment:
Perhaps this is a future story but if genesis is always a valid resume point then why include it at all when returning from `resumeFromStorage` and then check it with a test? Return [] instead, and when that happensis then the indexer can start from genesis. Saves from writing code both in the indexers and these tests.

Reply:
You're right. Will create a story for this.